src/Helper/DebitHelper.php

```diff
@@ -187,27 +187,29 @@ public function updateContactBank($orderId)
         ];
 
 
+
+
 //We dont want to save the contactId to till the ui for the customer bankdata is updated
 
-//        /** @var OrderRepositoryContract $orderContract */
-//        $orderContract = pluginApp(OrderRepositoryContract::class);
-//
-//        /** @var Order $order */
-//        // use processUnguarded to find orders for guests
-//        $order = $authHelper->processUnguarded(
-//            function () use ($orderContract, $orderId) {
-//                //unguarded
-//                return $orderContract->findOrderById($orderId, ['relation']);
-//            }
-//        );
-//        // Check whether the order truly exists in plentymarkets
-//        if(!is_null($order) && $order instanceof Order)
-//        {
-//            $contactId = $order->contactReceiver->id;
-//            if($contactId>0){
-//                $bankData['contactId'] = $contactId;
-//            }
-//        }
+        /** @var OrderRepositoryContract $orderContract */
+        $orderContract = pluginApp(OrderRepositoryContract::class);
+
+        /** @var Order $order */
+        // use processUnguarded to find orders for guests
+        $order = $authHelper->processUnguarded(
+            function () use ($orderContract, $orderId) {
+                //unguarded
+                return $orderContract->findOrderById($orderId, ['relation']);
```

Comment:
Müsste das nicht `'relations'` heissen?
```suggestion
                return $orderContract->findOrderById($orderId, ['relations']);
```

Reply:
Oh, im `OrderLegacyRepository` ist diese Relation ausgenommen!
@raphael-adam @MKanitz Wieso ist es noch so?
![Bildschirmfoto 2020-02-28 um 09 53 41](https://user-images.githubusercontent.com/9987759/75525596-6a712d00-5a10-11ea-8b8a-af0777844c96.png)